alignak_backend/timeseries.py

```diff
@@ -77,7 +77,19 @@ def prepare_data(item):
             m = re.search(r'^(.*)\.[\d]{10}$', fields['name'])
             if m:
                 fields['name'] = m.group(1)
-            fields['name'] = fields['name'].replace(" ", "_")
+
+            # Sanitize field name for Graphite:
```

Comment:
AFAIK, Influxdb do not have such constraints on field names, as such it is not necessary to change the fields names